<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GradeWindow</class>
 <widget class="QMainWindow" name="GradeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>年级设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rc.qrc">
    <normaloff>:/ico/images/appstu.ICO</normaloff>:/ico/images/appstu.ICO</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tbGrade">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>281</width>
      <height>161</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>年级编号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>年级名称</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>170</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>年级名称：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editID">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>170</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>170</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>年级编号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editName">
    <property name="geometry">
     <rect>
      <x>218</x>
      <y>170</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>201</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEdit">
    <property name="geometry">
     <rect>
      <x>107</x>
      <y>201</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDel">
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>201</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>201</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../rc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnExit</sender>
   <signal>clicked()</signal>
   <receiver>GradeWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editID</sender>
   <signal>returnPressed()</signal>
   <receiver>editName</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editName</sender>
   <signal>returnPressed()</signal>
   <receiver>btnAdd</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbGrade</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>btnEdit</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
